config/locales/it.yml

```diff
@@ -42,6 +42,8 @@ it:
       current_scope: "Contesto selezionato:"
       current_filters: "Filtri attivi:"
       no_current_filters: "Nessuno"
+    scopes:
+      all: "Tutti"
```

Comment:
@fuzziness just to maintain consistency, can you please move this `scopes:` block so it is listed before the `search_status:` one?

Reply:
fixed & added other missing